### PR TITLE
added mossaik

### PR DESCRIPTION
**Pull request by gabrielperales**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [x] you only add one (!) new helper per pull request.
- [x] you have checked if an open PR already exists.
- [x] the submitted website is focused on a single, development related issue.
- [x] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

**Comment by stefanjudis** [comment]
Thanks. This looks cool!

Could you check two things before merging?

1) Tiles doesn't seem to work in Firefox

grafik

2) When I play around with the app in FF, it'll break eventually

grafik

It looks like there's a problem with too many updates using the history API. :) 

Thanks!

**Comment by gabrielperales** [comment]
@stefanjudis does it work for you? Thanks again

File: helpers/mossaik.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Mossaik",
+  "desc": "Free SVG pattern generator. Create stunning shapes like waves, tiles, and blobs.",
+  "url": "https://mossaik.app/",
```
Comment (stefanjudis) [inline helpers/mossaik.json:4]:
```suggestion
  "url": "https://mossaik.app/app/",
```

File: helpers/mossaik.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Mossaik",
+  "desc": "Free SVG pattern generator. Create stunning shapes like waves, tiles, and blobs.",
```
Comment (stefanjudis) [inline helpers/mossaik.json:3]:
```suggestion
  "desc": "Create stunning shapes like waves, tiles, and blobs.",
```

**Comment by stefanjudis** [comment]
@gabrielperales Yes, looks great. Thank you!

Note: _pinging again after one day is a bit early in open source world_. 😉

**Pull request merged**